LICENSE

```diff
@@ -1,21 +1,13 @@
-The MIT License (MIT)
+   Copyright 2018 Udacity
 
-Copyright (c) 2018 Dineshkumar Pandiyan <flexdinesh@gmail.com>
+   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
ultimately we decided to use this because it is what the base fluent package is licensed under, and so it made sense to use the same one since they will be used together.